### PR TITLE
Case insensitive

### PR DESCRIPTION
**Pull request by osheroff**
@vanchi-zendesk

### CODE REVIEW

File: src/main/java/com/zendesk/maxwell/schema/Database.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public Database(String name, String encoding) {
 
 	public Table findTable(String name) {
 		for ( Table t: this.tableList ) {
```
Comment (vanchi-zendesk) [inline src/main/java/com/zendesk/maxwell/schema/Database.java]: probably better to do name = name.toLowerCase() here?

File: src/main/java/com/zendesk/maxwell/schema/Table.java [hunk 2]
```diff
@@ -50,15 +50,16 @@ private void initColumnOffsetMap() {
 		int i = 0;
 
 		for(ColumnDef c : columnList) {
-			this.columnOffsetMap.put(c.getName(), i++);
+			this.columnOffsetMap.put(c.getName().toLowerCase(), i++);
```
Comment (vanchi-zendesk) [inline src/main/java/com/zendesk/maxwell/schema/Table.java]: Is this to protect against existing entries? I see column defined with lowercase name always below.

**Comment by vanchi-zendesk** [comment]
You might want some tests for this one. It's not clear how the existing data is saved, and how all that will interact with this code.. (Or if the check was missed somewhere.)

File: src/main/java/com/zendesk/maxwell/schema/Table.java (hunk 2 above)
Comment (osheroff) [inline src/main/java/com/zendesk/maxwell/schema/Table.java]: ah, good point, that's not necessary.

**Comment by osheroff** [comment]
this is actually just a mess, https://dev.mysql.com/doc/refman/5.0/en/identifier-case-sensitivity.html and I suppose I could code for it, but, eh....

**Pull request closed without merging**